quic/s2n-quic-events/events/connection.rs

```diff
@@ -307,3 +307,10 @@ struct PacingRateUpdated {
     burst_size: u32,
     pacing_gain: f32,
 }
+
+#[event("recovery:bbr_state_changed")]
+/// The BBR state has changed
+struct BbrStateChanged {
```

Comment:
Can these states be reused for BBR 1? If not we might want to specify v2 here, yeah?

Reply:
Yeah, BBR1 has the same states, although it doesn't name the cycle phases. If we were to implement it I would just name the cycle phases the same and reuse this